LICENSE

```diff
@@ -671,4 +671,4 @@ into proprietary programs.  If your program is a subroutine library, you
 may consider it more useful to permit linking proprietary applications with
 the library.  If this is what you want to do, use the GNU Lesser General
 Public License instead of this License.  But first, please read
-<http://www.gnu.org/philosophy/why-not-lgpl.html>.
\ No newline at end of file
+<http://www.gnu.org/philosophy/why-not-lgpl.html>.
```

Comment:
This change (commit 27bee469d430a4f940dfacf69180e53a723fe284) looks like it accidentally got into the PR, but is somehow unrelated, right?